docs/Agent/config/config.md

```diff
@@ -0,0 +1,22 @@
+### Configuration
+
+**Note**: This Documentation is only required for administrative purposes.
+By Default the collected results are moved to pbench-server.
+
+
+#### Configure with Server < 1.0
+
+For the purpose of defining the required configuration, a pbench-agent configuration file must be created. The config file and the ssh key file must be present in the specified place for the ansible roles to function.
```

Comment:
For review purposes, if nothing else, keeping `.md` file lines to under 80 columns would be nice. The GitHub view wraps this line, but there's a single line to select and comment on, which can make things awkward. The MarkDown processor doesn't care about hard line wraps, but it can make things easier on lowly humans.

"For the purpose of defining the required configuration" is a very unwieldy sentence. In fact, I'd get rid of the first sentence and clarify the context of the second as something like "To integrate with a 0.69 Pbench Server, a Pbench Agent configuration needs the address of the Pbench Server and a private RSA key trusted by the Pbench Server `pbench` account." If you want to talk about Ansible installation, we're really into direct RPM installation, whereas I think we're hoping to encourage more people to use the Pbench Agent containers.